index.js

```diff
@@ -150,25 +144,6 @@ function onExecute(env) {
     exit(0);
   }
 
-  if (env.config.flags.verify) {
-    var pkgPath = env.config.flags.verify !== true ? env.config.flags.verify : 'package.json';
-    /* istanbul ignore else */
-    if (path.resolve(pkgPath) !== path.normalize(pkgPath)) {
-      pkgPath = path.join(env.cwd, pkgPath);
-    }
-    log.info('Verifying plugins in ' + pkgPath);
-    return getBlacklist(function(err, blacklist) {
```

Comment:
@sttk please also remove `getBlacklist` and the http utilities. Those should not be needed anymore.

Reply:
@phated Sorry. I forgot to remove `blacklist` directory and files included in it. I’ve removed them.